src/interface/blas1_interface.hpp

```diff
@@ -219,16 +219,30 @@ typename sb_handle_t::event_t _asum(
  * the platform being compiled for and other parameters, provides different
  * template parameters to ensure the most optimal kernel is constructed.
  *
- * @tparam localSize  specifies the number of threads per work group used by
- *                    the kernel
- * @tparam localMemSize specifies the size of local shared memory to use, which
+ * @tparam localSize    Specifies the number of threads per work group used by
+ *                      the kernel
+ * @tparam localMemSize Specifies the size of local shared memory to use, which
  *                      is device and implementation dependent. If 0 the
  *                      implementation use a kernel implementation which doesn't
  *                      require local memory.
+ * @tparam usmManagedMem Specifies if usm memory allocation is automatically
+ *                       managed  or not. The memory automatically managed
+ *                       requires that atomic address space is set to generic.
+ *                       This is a strict requirement only for AMD gpus, since
+ *                       otherwise it will rely on pcie atomics
+ *                       which we cannot enforce, guarantee or check due to its
+ *                       hardware nature. Other targets do not have the same
+ *                       strong dependency and managed memory is handled
+ *                       correctly in any case by default. It is automatically
+ *                       initialized to false to reduce verbosity of
+ *                       initialization for many targets since only one of them,
+ *                       with specific allocation type, requires a different
+ *                       value. Having a default value allows the compiler to
+ *                       handle automatically other templates.
```

Comment:
```suggestion
 * @tparam usmManagedMem Specifies if usm memory allocation is automatically
 *                       managed  or not. Automatically managed memory
 *                       requires that atomic address space is set to generic.
 *                       This is a strict requirement only for AMD GPUs, since
 *                       AMD's implementation of atomics may depend on specific
 *                       hardware configurations (PCIe atomics) that cannot be checked at runtime.
 *                       Other targets do not have the same
 *                       strong dependency and managed memory is handled
 *                       correctly by default.
```

I don't think you need to justify your choice of having a default in the documentations.

Reply:
Thank you @hjabird I am still new to writing documentation. I will fix it.

Reply:
Fixed in 5713422f46b3050266aaf6b705d42d47f72e7310